main.tex

```diff
@@ -341,7 +341,7 @@ \section{Example}
     # use astropy's wcsaxes tool to create an RA/Dec image
     ax = fig.add_axes([0.15, 0.1, 0.8, 0.8], projection=mywcs)
     ax.set_xlabel("RA")
-    ax.set_ylabel("Dec")
+    ax.set_ylabel("DEC")
```

Comment:
:-1: to this.  RA is capitalized because it's an acronym.  Dec is not capitalized because it's an abbreviation.

Reply:
(so could you revert this change, then I'll merge the PR?)